subworkflows/local/abacas_multifasta.nf

```diff
@@ -0,0 +1,63 @@
+//
+// Process multiple multifasta files with abacas
+//
+include { ABACAS        } from '../../modules/nf-core/abacas/main'
+
+workflow ABACAS_MULTI {
+    take:
+    scaffold   // channel: [ val(meta), path(scaffold) ]
+    multifasta // channel: /path/to/genome.fasta
+    assembler
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    //
+    // Split multifasta file into individual fasta files
+    //
+    multifasta
+        .splitFasta( by: 1, file: true )
+        .set { ch_fasta }
+
+    //
+    // Run abacas on each fasta file
+    //
+    ch_abacas = Channel.empty()
+
+    scaffold
+        .combine (ch_fasta)
+        .set { ch_scaffold_fasta }
+
+    ABACAS (
+        ch_scaffold_fasta.map { meta, scaffold, fasta -> tuple( meta, scaffold ) },
+        ch_scaffold_fasta.map { meta, scaffold, fasta -> fasta }
+    )
+    ch_abacas = ABACAS.out.results
+    ch_versions = ch_versions.mix(ABACAS.out.versions)
+
+    //
+    // Concatenate abacas results
+    //
+
+    ch_abacas
+        .map { meta, files -> tuple(meta, files[3]) }
+        .multiMap{ meta, fasta ->
+            metadata: [meta.id, meta.clone()]
```

Comment:
`meta.clone()` is not really good pratice. That's on me. Think you should drop `.clone()`